Backend/app/rider-platform/rider-app/Main/spec/Storage/Booking.yaml

```
@@ -223,6 +223,7 @@ Booking:
     isReferredRide: Maybe Bool
     disabilityTag: Maybe Text
     journeyLegOrder: Maybe Int
+    isDeleted : Maybe Bool
```

yamllint (1.35.1)

[warning] too many spaces before colon

(colons)